### ISSUE TITLE
:bug: Special day: Duckbot inception day --- Wrong alive time

### ISSUE THREAD

**Issue by jameshughes89** [duck-dynasty/duckbot#640]
[Within `announce_day/special_days.py`, duckbot says it's alive time in seconds incorrectly. ](https://github.com/duck-dynasty/duckbot/blob/2cae4ecc4402812cb9067ae6e2f10d538bb250c0/duckbot/cogs/announce_day/special_days.py#L52)

Right now it uses the `seconds` attribute, but it [turns out the `seconds` attribute has a range of 0 --- 86399s which is 1 day. ](https://docs.python.org/3/library/datetime.html#timedelta-objects)

[Instead, the `total_seconds()` method would do the trick as it returns the total number of seconds from all resolutions of time in the datetime. ](https://docs.python.org/3/library/datetime.html#datetime.timedelta.total_seconds)

In other words

Change 

```python
(now()-datetime(2020, 12, 3, 10, 39, tzinfo=timezone())).seconds
```
to 

```python
(now()-datetime(2020, 12, 3, 10, 39, tzinfo=timezone())).total_seconds()
```

**Comment by twentylemon** [comment]
There's even a [test for this](https://github.com/duck-dynasty/duckbot/blob/2cae4ecc4402812cb9067ae6e2f10d538bb250c0/tests/cogs/announce_day/special_days_test.py#L41), but it makes the same mistake. That value should probably be hard coded in the test rather than calculated.

**Comment by twentylemon** [comment]
Further, the code uses `now() - $commit_time`, which is probably wrong since `year` is a parameter to the special days list. The `now` calculation should use that year as input somehow.

**Comment by jameshughes89** [comment]
> Further, the code uses `now() - $commit_time`, which is probably wrong since `year` is a parameter to the special days list. The `now` calculation should use that year as input somehow.

I need clarification on this.
